flash-lso/src/lib.rs

```diff
@@ -1,5 +1,5 @@
 //! Library for reading and writing the Adobe Flash Local Shared Object (LSO) file format and the contained AMF0/AMF3 data
-
+#![type_length_limit = "94603681"]
```

Comment:
Do we really have a 100M (whatever) long type here? :skull: So it's not just rustc nightly being funky?
Is this also why this crate is so slow to build? :thinking:

Reply:
It's 100% an actual real thing, as far as I can tell. And it's a new rustc intended thing that they're limiting the type length: https://github.com/rust-lang/rust/pull/125507

Reply:
Removed thanks to #64

Reply:
Imposing a limit is fine, especially one this large (whatever the unit is, I'm assuming bytes). But why would such a lorge type be necessary?! 😱 (No need to bother answering, I'm just mildly shocked.)

Reply:
No idea! It needs to be that large specifically (give or take a few percent). I had to keep increasing it until it worked 😭